az-core/src/main/java/azkaban/Constants.java

```diff
@@ -688,5 +690,9 @@
         + ".request";
 
     public static final String FLOW_PARAM_POD_ENV_VAR = "pod.env.var.";
+
+    // Constant to allow test version to be passed as flow parameter. Passing test version will be
+    // allowed for Azkaban ADMIN role only
+    public static final String FLOW_PARAM_ALLOW_TEST_VERSION = "allow.test.version";
```

Comment:
Shall we rename it to allow.pod.test.version?

Reply:
With allow.pod.test.version does not meaning someting like pod test version. How about allow.image.test.version? I think this name provides more context of the usage.

Reply:
That sounds better! Thanks